content/04.model-complexity.regularization.md

```diff
@@ -0,0 +1,55 @@
+(2) LASSO regression on the other hand works well for selecting few important features since its effect can shrink some features more than the others. 
+Thus it helps in selecting most important features while irrelevant features are lost and shrunk to 0. This selection attribute of LASSO (in a sample set of size "n", LASSO can only select "n" features for the model) may be an advantage in reducing model complexity, but a disadvantage in cases where identification of all possible collinear features is important (eg. all biomarkers correlating to a particular disease phenotype) [@doi:10.1038/nmeth.4014]. 
+(3) Elastic net regression is a combination of LASSO and ridge regression[@doi:10.1111/j.1467-9868.2005.00503.x]. 
+Both of the methodologies when applied together helps to select most useful features, specially where there are a lot of correlated features. 
+In this setup, LASSO leads to selection of one of the correlated features and reduces the others to 0 (grouping of features). 
+The ridge regression aspect of elastic-net then minimizes the magnitude of the features that were selected without shrinking any of them to 0 (minimization). 
+
+Any supervised learning implementation in rare disease would require robustness towards feature selection from a small number of samples. 
+This robustness is mostly acquired through the combination of various regression strategies. 
+Since machine learning applications in rare disease is infrequent, combination strategies used for rare variant discovery and immune cell signature discovery can serve as good case studies to examine. 
+Many deleterious genomic variants can be extremely rare due to the constant selection pressure working against them. 
+Since the frequency of a rare variant is so low (less than 1%) applying routine statistical procedures that were extensively developed for common variant association, to analyze a low minor allele frequency (MAF) seem inappropriate [@doi:10.1038/nrg2867]. 
+For its feature selection attribute, LASSO or least absolute shrinkage and selection operator has been widely applied in microarray and GWAS data for common variants. 
+But since LASSO by itself is too stringent for rare variants, it has been employed along with group penalties to help identify rare variants/ low frequency predictors [@doi:10.1093/bioinformatics/btq448]. 
+Variations of LASSO have also been implemented to aggregate or group the occurrance of rare variants together by gene or chromosome location [@doi:10.1002/gepi.21746; @doi:10.1186/1753-6561-5-S9-S100; @doi:10.1016/j.ajhg.2008.06.024]. 
+In this strategy, a 0–1 dummy variable was created for each SNP based on the presence or absence of the rare variant. 
+Then linear combinations of the selected dummy variables were considered by using the LASSO procedure. 
+Even though most of the dummy variables were 0, their linear combination was more likely to be nonzero thus leading to increased signal to noise ratio for the rare variants. 
```

Comment:
I'm not sure if everyone in the intended audience will know the term dummy variable (possibly related to #24 depending on the scope of that "box").

Reply:
added to issue #50